SecurityPkg/Tcg/Tcg2Smm/Tcg2Smm.c

```diff
@@ -86,6 +86,13 @@ TpmNvsCommunciate (
   switch (CommParams->Function) {
     case TpmNvsMmExchangeInfo:
       DEBUG ((DEBUG_VERBOSE, "[%a] - Function requested: MM_EXCHANGE_NVS_INFO\n", __FUNCTION__));
+      // MU_CHANGE TCBZ4378 [BEGIN] - Check for invalid NVS buffer location
+      if (IsBufferOutsideMmValid (CommParams->TargetAddress, sizeof (TCG_NVS))) {
+        DEBUG ((DEBUG_ERROR, "[%a] - NVS buffer in invalid location!\n", __FUNCTION__));

```

Comment:
For new code, `__func__` will blend better with the convention currently used in the file in edk2.

```suggestion
        DEBUG ((DEBUG_ERROR, "[%a] - NVS buffer in invalid location!\n", __func__));

```